### PR TITLE
refactored Jotform101 page

### PR DESCRIPTION
**Pull request by dhaladitya108**

### CODE REVIEW

File: components/competitors/PopularUseCase.vue [hunk 1]
```diff
@@ -40,6 +41,10 @@ export default {
       type: Array,
       required: true,
     },
+    numberOfCol: {
+      type: Number,
+      default: 4,
```
Comment (piyushsinghania) [inline components/competitors/PopularUseCase.vue:46]: ideally default should not be `4`

File: pages/jotform-101.vue [hunk 2]
```diff
@@ -49,23 +61,25 @@
 </template>
 
 <script>
-import KeyFeaturesOfJotform from '../components/jotform101/key-features-of-jotform.vue'
-import WhatMakesJotformStandOut from '../components/jotform101/what-makes-jotform-stand-out.vue'
-import JotformPopularUseCase from '../components/jotform101/popular-use-case.vue'
-import JotformLimitations from '../components/jotform101/limitations.vue'
+import JotformKeyFeatures from '../components/competitors/KeyFeatures.vue'
```
Comment (piyushsinghania) [inline pages/jotform-101.vue:64]: Why not just name it `KeyFeatures`?

**Review by piyushsinghania — APPROVED** [review]

File: pages/jotform-101.vue (hunk 2 above)
Comment (dhaladitya108) [reply pages/jotform-101.vue:64]: I was thinking as we are using it on the jotform page let's name it JotformKeyFeatures. 
Yes you are right naming it only KeyFeatures would be appropriate here.

File: components/competitors/PopularUseCase.vue (hunk 1 above)
Comment (dhaladitya108) [reply components/competitors/PopularUseCase.vue:46]: It should be 4. It's actually the bootstrap col no. 
I will rename it so, that it's not confusing.

**Pull request merged**